src/openrct2-ui/interface/ViewportInteraction.cpp

```diff
@@ -253,7 +253,8 @@ InteractionInfo ViewportInteractionGetItemRight(const ScreenCoordsXY& screenCoor
     if ((gScreenFlags & SCREEN_FLAGS_TRACK_DESIGNER) && gS6Info.editor_step != EditorStep::RollercoasterDesigner)
         return info;
 
-    auto flags = static_cast<int32_t>(~EnumsToFlags(ViewportInteractionItem::Terrain, ViewportInteractionItem::Water));
+    constexpr auto flags = static_cast<int32_t>(
```

Comment:
Which static checker flagged the constexpr? Sounds interesting

Reply:
The one that comes with VS by default.